File-Formats/DocIO/Working-with-Find-and-Replace.md

```diff
@@ -884,6 +884,97 @@ document.Close();
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Find-and-Replace/Replace-misspelled-word).
 
+## Find and replace non-breaking spaces with regular spaces
```

Comment:
@sharonaumao - Please change the description like below, 
"You can find non-breaking spaces used in a Word document and replace them with regular spaces."